### DISCUSSION TITLE
Projects beta - Persist a preferred layout of a view

### DISCUSSION THREAD

**Discussion by pana-cc** [community/community#6370] [Projects and Issues] [answered]
I have a project with two views. One is a Kanban board which I want to stay in board layout. Another is for planning where I want to always keep the issues in a grid, it shows additional columns for estimates, etc. Currently every time I open the default Kanban board, I have to manually switch from table to board. It would be nice if I could set a preferred/default layout for a view.

**Comment by pana-cc** [comment, marked as answer]
Oh I got it now. Once I rearrange them a blue dot appears and I have to go click "save changes"

**Reply by Flibben** [reply]
Wow, thanks for pointing this out. Glad I found your answer, this was driving me crazy!
